starter_code/hangman.js

```diff
@@ -27,11 +27,13 @@ let hangmanCanvas;
       }
 
    checkClickedLetters(key) {
```

Comment:
aqui un codigo asi pasa los test de jasmine, pero estabas cerca 🙃

  ` if (this.letters.includes(key)){`
     ` return false`
 `   } else {`
    `  return true`
 `   }`